transformer_engine/jax/flax/module.py

```diff
@@ -171,6 +178,11 @@ def __call__(self, inputs: Array, mask: Array = None, bias: Array = None) -> jnp
 
 class LayerNorm(nn.Module):
     r"""
+    .. warning::
```

Comment:
This warning is not needed with that change to added the deprecated function, right?

Reply:
Done.